### DISCUSSION TITLE
Lemmatization for Indonesian Language support

### DISCUSSION THREAD

**Discussion by k4waiiiii** [explosion/spaCy#9760] [Language Support]
Lemmatization for Indonesian language is removed? I didn't see `lemmatizer.py`

**Comment by polm** [comment]
Support for Indonesian lemmatization has not been removed. In order to use the lemmatizer you need to install `spacy-lookups-data` add the lemmatizer to your pipeline like this:

```
import spacy
nlp = spacy.blank("id")
nlp.add_pipe("lemmatizer", config={"mode": "lookup"})
nlp.initialize()
```

This has been the case as long a v3 has been out, if this is a change from what you expect I guess you're coming from v2? See [the docs](https://spacy.io/usage/linguistic-features#lemmatization) for notes on the change.

**Reply by k4waiiiii** [reply]
I thought it was deleted but it didn't. Thank you for answering my question.
